src/executors/collect_battery.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Created by xiaofei.lan@rokid.com on 2019/04/26.
+ */
+
+#ifndef YODA_SIXSIX_COLLECT_BATTERY_H
+#define YODA_SIXSIX_COLLECT_BATTERY_H
+
+#include "executor_def.h"
+
+#define CHARGER_PATH "/sys/class/power_supply/bq25890-charger"
+#define BATTERY_PATH "/sys/class/power_supply/battery"
+
+#define GET_CHARGER_FILE(path) CHARGER_PATH path
+#define GET_BATTERY_FILE(path) BATTERY_PATH path
+
+using namespace std;
```

Comment:
It is not recommend to using namespace in a header file, this will cause other header files to be affected by the contamination